hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/snapshot/TestOmSnapshotUtils.java

```diff
@@ -66,7 +66,7 @@ public void testLinkFiles(@TempDir File tempDir) throws Exception {
     assertFalse(tree2.exists());
     assertFalse(f1Link.exists());
 
-    OmSnapshotUtils.linkFiles(tree1, tree2);
+    OmSnapshotUtils.linkFilesOrCopy(tree1, tree2);
```

Comment:
Please add similar test case for copy case, when both files are in different partitions. You can achieve this by mocking the layer.

Reply:
Mocking this isSamePartition function should help. From what I know you cannot mock Files interface through mockito but PowerMockito can do it.